openthread-tock/platform/platform-tock/misc.c

```diff
@@ -0,0 +1,18 @@
+// TODO COPYRIGHT
+
+#include <openthread/platform/misc.h>
+
+void otPlatReset(otInstance *aInstance) {
+  // TODO
```

Comment:
You can `exit()`? Tock can be set to reset apps on exit

Reply:
This is a good idea! (I'm planning to implement these todos in a follow up PR and will look further into this then)